src/pages/Home/Home.jsx

```diff
@@ -1,15 +1,25 @@
-import React from 'react';
-import './Home.scss';
-import PokeList from '../../PokeList/PokeList';
-import Welcome from '../../Welcome/Welcome';
+import React, {useContext} from "react";
+import "./Home.scss";
+import PokeList from "../../PokeList/PokeList";
+import Welcome from "../../Welcome/Welcome";
+import Spinner from "../../Spinner/Spinner";
+import useFetchPokemon from "../../PokeList/useFetchPokemon";
+import { PokemonContext } from "../../context/PokemonContext";
 
 const Home = () => {
+  const {state: { isFetching, pokemonList }} = useContext(PokemonContext);
+  useFetchPokemon();
   return (
-    <div className='home-page'>
-        <Welcome />
-        <PokeList />
+    <div className="home-page">
+      {isFetching && <Spinner />}
```

Comment:
In addition de isFetching, also it should be verified if the length of pokemonList is equal to 0, because if only isFetching variable is considered, when lazy loading is implemented, the spinner will be shown

Reply:
Fixed